### ISSUE TITLE
A complex parameter using content in paths validator will failed

### ISSUE THREAD

**Issue by zghnr1993** [getkin/kin-openapi#56]
if i defined following openapi , when i validator it ,will failed !
```
in: query
name: coordinates
content:
  application/json:
    schema:
      type: object
      required:
        - lat
        - long
      properties:
        lat:
          type: number
        long:
          type:
```

i think this part need to follow "resolveRequestBodyRef" function . add 
````
for _, contentType := range value.Content {
		for name, example := range contentType.Examples {
			if err := swaggerLoader.resolveExampleRef(swagger, example, path); err != nil {
				return err
			}
			contentType.Examples[name] = example
		}
		if schema := contentType.Schema; schema != nil {
			if err := swaggerLoader.resolveSchemaRef(swagger, schema, path); err != nil {
				return err
			}
		}
	}
````

in resolveParameterRef function !

**Comment by fenollp** [comment]
Hello @zghnr1993 ! Pull requests and tests are welcomed!

**Comment by fenollp** [comment]
cc https://github.com/getkin/kin-openapi/pull/57

**Comment by fenollp** [comment]
Fixed by #57
